core/src/smtp/parser.rs

```diff
@@ -111,6 +111,8 @@ pub fn is_full_inbox(e: &str) -> bool {
 	|| e.contains("over quota")
 	// 550 user has too many messages on the server
 	|| e.contains("too many messages")
+	// https://newsroom.gmx.net/2017/09/28/unzustellbar-fehlermeldungen-email/
+	|| e.comtains("exceeded storage allocation")
```

Comment:
contains, not comtains.

Reply:
any updates on this? I'll merge once fixed